src/test/java/com/sap/sgs/phosphor/fosstars/data/github/UsesGitHubPlatformTest.java

```diff
@@ -0,0 +1,120 @@
+package com.sap.sgs.phosphor.fosstars.data.github;
+
+import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.USES_GITHUB_PLATFORM;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+import com.sap.sgs.phosphor.fosstars.model.value.ValueHashSet;
+import com.sap.sgs.phosphor.fosstars.tool.github.GitHubProject;
+import com.sap.sgs.phosphor.fosstars.tool.github.GitHubProjectValueCache;
+import java.io.IOException;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GitHub;
+
+public class UsesGitHubPlatformTest {
+
+  @Test
+  public void testUsesGitHubProjectFalse() throws IOException {
+    GitHub github = mock(GitHub.class);
+    UsesGitHubPlatform provider = new UsesGitHubPlatform(github);
+    provider = spy(provider);
+    when(provider.cache()).thenReturn(new GitHubProjectValueCache());
+
+    GHRepository repository = mock(GHRepository.class);
+    when(github.getRepository(any())).thenReturn(repository);
+
+    when(repository.getMirrorUrl()).thenReturn("http://git.test");
```

Comment:
If I understand correctly, `getMirrorUrl()` returns a link to the main repository. If so, could you please use `https://other.scm.com/org/original_repository'. And then please don't use `spring-integration` but replace it with `new GitHubProject("org", "mirror_repository")`.